HighTeach/urls.py

```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homePage, name='homePage')
+    path('', views.homePage, name='homePage'),
+    path('logout/', views.sign_out, name='sign out')
```

Comment:
```suggestion
    path('', views.homePage, name='home_page'),
    path('logout/', views.sign_out, name='sign_out')
```
I think URL names should be in snake case

Reply:
This is a temporary change in this PR till we have the loggedInPage, you will see that a change was made in this [PR](https://github.com/redhat-beyond/HighTeach/pull/82) after adding loggedInPage.html

Reply:
As you can see here [link](https://docs.djangoproject.com/en/4.2/topics/http/urls/#naming-url-patterns).
The string used for the URL name can contain any characters you like.
I think it is better that we decide as a group on a convention for writing the path names

Reply:
I guess it can be kept as is for now until we decide on a convention.